Pythonwin/pywin/idle/readme.txt

```diff
@@ -6,7 +6,8 @@ Pythonwin.  In ALL cases, the files in this directory that also appear
 in the main IDLE directory should be indentical to the latest available
 for IDLE.  
 
-Eg, If you have Python 1.5.2 installed, the files in this
+<!-- TODO: Is this still accurate? -->
```

Comment:
I think this should be reworded to reflect something like "if the files in this directory are newer than the ones shipped with your Python version" or similar.

Reply:
I'm not sure how I would integrate that text. If they are newer, then what?

The current text seems to imply that this directory was based on IDLE from a specific python version (>3.7). What I'm not sure about, is whether saying that Python 3.8 is the version it's based on is accurate. Or if I'm even understanding this message correctly.

Also by "later" does this text mean "newer" ?

Reply:
This comment was trying to reflect that the files in this dir also exist in IDLE, but may have been released as part of pywin32 before a Python release was made. So yeah, I guess this entire readme can now just say something like "these files have been forked from IDLE" or similar.

Reply:
Ok. I have pushed a wording change suggestion that should not fall out of date.